### DISCUSSION TITLE
Not able to see languages on my flutter apps repositories...

### DISCUSSION THREAD

**Discussion by VaishnaviNagrale** [community/community#45423] [New to GitHub]
I had recently created many flutter apps repositories and pasted flutter apps but in that they are not showing languages as generally they show as dart at bottom plz help...

**Comment by IagoAntunes** [comment]
If you run the application on a cell phone, the predominant language will be Dart, but if you run it on Windows, the predominant language may be C++, and on the Web the language is also different
